clients/docs/scala.rst

```diff
@@ -0,0 +1,175 @@
+.. _client-examples-scala:
+
+Scala
+======
+
+In this tutorial, you will run a Scala client application that produces messages
+to and consumes messages from an |ak-tm| cluster. For more information, please
+see the `application development documentation
+<https://docs.confluent.io/current/api-javadoc.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__
+
+.. include:: includes/client-example-overview.rst
+
+
+Prerequisites
+-------------
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. .. include:: includes/clients-checkout.rst
+
+#. Change directory to the example for Scala.
+
+   .. code-block:: bash
+
+      cd clients/cloud/scala/
+
+#. .. include:: includes/client-example-create-file-java.rst
+
+
+Basic Producer and Consumer
+---------------------------
+
+.. include:: includes/producer-consumer-description.rst
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+#. Run the example consumer:
+
+   .. code-block:: bash
+
+      cd examples/clients/cloud/scala
+
+#. Build the client example:
+
+   .. code-block:: bash
+
+      sbt clean compile
+
+#. Run the consumer:
+
+   .. code-block:: bash
+
+      sbt “runMain io.confluent.examples.clients.scala.Consumer $HOME/.confluent/java.config testtopic”
+
+   You should see:
+
+   .. code-block:: text
+
+         <snipped>
+
+         Polling
+         ....
+         <snipped>
+
```

Comment:
Need line to view scala Consumer source code

Reply:
added - thank you!